miri

```diff
@@ -133,6 +133,20 @@ find_sysroot() {
 }
 
 ## Main
+if [[ -z "$AUTO_OPS" ]]; then
```

Comment:
This entire block should probably be before `TOOLCHAIN=`. Otherwise, it might be the case that you start with toolchain A, then auto-toolchain runs and sets the local toolchain to Miri, but we will still use toolchain A.